### PR TITLE
Add READ permissions for mutating webhooks

### PR DESCRIPTION
**Pull request by nrfox**
Necessary for https://github.com/kiali/kiali/pull/7695

### CODE REVIEW

**Comment by jmazzitelli** [comment]
Similar to the [helm chart PR](https://github.com/kiali/helm-charts/pull/279/files) that grants the permission to the operator, you also need to add it to the OLM metadata, of which we have three. See the [checklist here](https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-operator-resource).

The three OLM metadata files to add this new permission are these (probably just add them to the bottom):
1. https://github.com/kiali/kiali-operator/blob/master/manifests/kiali-upstream/1.90.0/manifests/kiali.v1.90.0.clusterserviceversion.yaml#L533
2. https://github.com/kiali/kiali-operator/blob/master/manifests/kiali-community/1.90.0/manifests/kiali.v1.90.0.clusterserviceversion.yaml#L607
3. https://github.com/kiali/kiali-operator/blob/master/manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml#L643

Note: We are very likely going to rename the 1.90 CSVs to 2.0. But you can change the 1.90 ones and we'll do the rename in another PR when we prepare for the 2.0 release.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
see my comment above

File: manifests/kiali-upstream/1.90.0/manifests/kiali.v1.90.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -521,14 +521,15 @@ spec:
...
-        - apiGroups: ["route.openshift.io"]
-          resources:
-          - routes
-          verbs:
-          - get
         - apiGroups: ["authentication.k8s.io"]
           resources:
           - tokenreviews
           verbs:
-          - create
```
Comment (jmazzitelli) [inline manifests/kiali-upstream/1.90.0/manifests/kiali.v1.90.0.clusterserviceversion.yaml:533]: did this "create" verb get deleted by accident? I see tokenreviews still has "create" in other places and there seems to be an empty "verbs" section in here now.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

**Review by jmazzitelli — APPROVED** [review]
i didn't run it, but it looks good. jay said he's going to test the rest of this stuff, I'll defer to him.

**Review by jshaughn — APPROVED** [review]
Tested using operator and minikube.

**Comment by jmazzitelli** [comment]
This missed the kubernetes template for role-viewer. Molecule tests win again :) 

https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/kubernetes/role-viewer.yaml#L73
```
  - mutatatingwebhookconfigurations
```

**Pull request merged**